spec/requests/comments_spec.rb

```diff
@@ -270,4 +270,26 @@
   describe "PATCH /comments/:comment_id/unhide" do
     include_examples "PATCH /comments/:comment_id/hide or unhide", path: "unhide", hidden: "false"
   end
+
+  describe "DELETE /comments/:comment_id" do
+    before { sign_in user }
+
+    it "deletes a comment if the article is still present" do
+      delete "/comments/#{comment.id}"
+
+      expect(Comment.find_by(id: comment.id)).to be_nil
+      expect(flash[:notice]).to eq("Comment was successfully deleted.")
+      expect(response.location).to eq("http://www.example.com/#{user.username}/#{article.slug}")
```

Comment:
maybe we can parse the path with `Addressable::URI`, as this text would break if we ever change the fake test hostame

Reply:
Good point. Instead of parsing URLs I opted for the following:

```ruby
expect(response).to redirect_to(comment.commentable.path)

expect(response).to redirect_to(user_path(user))
```